src/test/ui/feature-gates/feature-gate-pin_macro.stderr

```diff
@@ -0,0 +1,21 @@
+error[E0658]: use of unstable library feature 'pin_macro'
```

Comment:
Is there a reason for this test?

We don't normally test that `#[unstable]` library features are actually unstable. If we'd do that for every unstable macro/type/function/etc., that would quickly add a lot of unnecessary tests, making everybody wait even longer when they wait for the ui tests.

Reply:
No reason besides my not knowing about it: what you've mentioned makes a lot of sense 👍